tests/testproject/TestProject.ServiceA/Program.cs

```diff
@@ -8,6 +8,8 @@
 var app = builder.Build();
 
 app.MapGet("/", () => "Hello World!");
+app.MapGet("/get-only", () => Results.Ok());
```

Comment:
Do we need to modify this? What uses this?

Reply:
Nothing, it was added as potential target for a test but I didn't end up using it. Will remove.